app/order/models.py

```diff
@@ -16,6 +16,7 @@ class Order(models.Model):
         'members.UserAddress',
         on_delete=models.CASCADE,
     )
+    status = models.CharField(max_length=1, help_text='주문 상태')
```

Comment:
https://docs.djangoproject.com/en/3.1/howto/writing-migrations/#migrations-that-add-unique-fields